spec/unit/show-hide-content.spec.js

```diff
@@ -30,6 +30,17 @@ describe('show-hide-content', function () {
         '<label>No</label>' +
         '</div>' +
         '<div id="show-hide-radios" class="panel js-hidden" />' +
+
```

Comment:
I presume it’s for readability — in the absence of multi-line strings, it’s a way to avoid all the HTML being on one line.